scipy/stats/_multivariate.py

```diff
@@ -5739,3 +5749,295 @@ def _sample_uniform_direction(dim, size, random_state):
     samples = random_state.standard_normal(samples_shape)
     samples /= np.linalg.norm(samples, axis=-1, keepdims=True)
     return samples
+
+
+class dirichlet_multinomial_gen(multi_rv_generic):
+    r"""A Dirichlet multinomial random variable.
+
+    Methods
+    -------
+    logpmf(alpha, x):
+        Log of the probability mass function.
+    pmf(alpha, x):
+        Probability mass function.
+    mean(alpha, n):
```

Comment:
I thought about whether `alpha` or `n` should come first. I think I agree with `alpha`.